lib/internal/fs/promises.js

```diff
@@ -22,6 +22,9 @@ const {
   SafeArrayIterator,
   Symbol,
   Uint8Array,
+  SymbolIterator,
+  SymbolAsyncIterator,
+  ArrayIsArray
```

Comment:
nit: ASCII order (please put `ArrayIsArray` above)
```suggestion
  SymbolAsyncIterator,
  SymbolIterator,
  Uint8Array,
```